extensions/applicationinsights-debugplugin-js/src/components/debugBins.ts

```diff
@@ -0,0 +1,116 @@
+export class DebugBinParent {
+  public el: HTMLDivElement;
```

Comment:
Do these need to be public? (if not then *perhaps* use DynamicProto() or just create your own closure in the constructor)